std_msgs/Header header
common/Uint8TimeStamped adcu_brk_active
common/Uint8TimeStamped adcu_brk_ctrlmode
common/FloatTimeStamped adcu_brk_tgtpedpos
common/FloatTimeStamped adcu_brk_tgtpress
common/FloatTimeStamped adcu_brk_tgtacc

